src/drivers/actuators/voxl_esc/voxl_esc_params.c

```diff
@@ -262,3 +262,17 @@ PARAM_DEFINE_INT32(VOXL_ESC_T_WARN, 0);
  * @max 200
  */
 PARAM_DEFINE_INT32(VOXL_ESC_T_OVER, 0);
+
+
+/**
+ * GPIO Control Channel
+ *
+ *
+ * @reboot_required true
+ *
+ * @group VOXL ESC
+ * @value 0 - Disabled
+ * @min 0
+ * @max 6
+ */
+PARAM_DEFINE_INT32(GPIO_CTL_CH, 0);
```

Comment:
I would keep some kind of prefix so it doesn't get confused or potentially conflict with something system wide.

Could it simply be `VOXL_ESC_GPIO`?

Reply:
@dagar I changed the name to VOXL_ESC_GPIO_CH